content/docs/context.md

```diff
@@ -141,6 +141,12 @@ Changes are determined by comparing the new and old values using the same algori
 
 ```js
 class MyClass extends React.Component {
+  constructor(props, context) {
```

Comment:
Instead of using `context` as a parameter, the `context` should be accessed using the `contextType`. Please see: https://react.dev/reference/react/Component#context